.nvim/toggletasks.json

```diff
@@ -0,0 +1,19 @@
+{
+    "tasks": [
+        {
+            "name": "build",
+            "cwd": "${lsp_root}",
+            "cmd": "meson compile -C build"
+        },
+        {
+            "name": "install",
+            "cwd": "${lsp_root}",
+            "cmd": "sudo meson install -C build"
+        },
+        {
+            "name": "run",
+            "cwd": "${lsp_root}",
+            "cmd": "meson compile -C build && meson install -C build && ./target/bin/rnote"
+        }
+    ]
+}
```

Comment:
Oh eh, you might be wondering why this is here. This is my config for [toggletasks.nvim](https://github.com/jedrzejboczar/toggletasks.nvim) to quickly compile and run rnote. If you want, I can remove it.

Reply:
please do.

Reply:
done! :)